packages/fxa-customs-server/lib/email_record.js

```diff
@@ -250,7 +249,7 @@ module.exports = function (limits, now) {
 
   EmailRecord.prototype.isOverPasswordResetOtpLimit = function () {
     this.trimPasswordResetOtps(now());
-    return this.os.length > limits.maxPasswordResetOtpEmails;
+    return this.os.length >= limits.maxPasswordResetOtpEmails;
```

Comment:
Does this need the `=`? I would think it needs to be over the limit of 5 to get triggered.